watertap/examples/chemistry/tests/test_pure_water_pH.py

```diff
@@ -142,6 +142,7 @@ def base_units():
 def solver():
     s = get_solver()
     s.options["max_iter"] = 200
+    s.options["tol"] = 1e-12
     return s
```

Comment:
It was, but I think I've found a global set of Ipopt options that work.

This one was strange -- a `tol` of `1e-06` "works" (the tests pass). However, smaller values (e.g., `1e-07`, `1e-09`, would cause regression test failures. Looking at the log, Ipopt happens to get lucky and with `1e-06` solves to a tighter tolerance than `1e-09` due to when the solver chooses to kick out of restoration.